content/lib/pencil-tracer.js

```diff
@@ -946,6 +1014,28 @@
       return results;
     };
 
+    JavaScriptInstrumenter.prototype.substringByLocation = function(loc) {
+      var j, lineNum, ref, ref1, result;
+      result = "";
+      for (lineNum = j = ref = loc.start.line, ref1 = loc.end.line; ref <= ref1 ? j <= ref1 : j >= ref1; lineNum = ref <= ref1 ? ++j : --j) {
+        result += lineNum === loc.start.line && lineNum === loc.end.line ? this.lines[lineNum].slice(loc.start.column, loc.end.column) : lineNum === loc.start.line ? this.lines[lineNum].slice(loc.start.column) : lineNum === loc.end.line ? this.lines[lineNum].slice(0, loc.end.column) : this.lines[lineNum];
+      }
+      return result;
+    };
+
+    JavaScriptInstrumenter.prototype.argsToString = function(argNodes) {
+      var arg;
```

Comment:
Move this inside, to where it's needed. (Yes, I know that Javascript will effectively move the declaration out here.)